src/include/storage/index/index.h

```diff
@@ -58,12 +57,11 @@ class Index {
 
   /**
    * Creates a new index wrapper.
-   * @param oid identifier for the index
    * @param constraint_type type of index
    * @param metadata index description
    */
-  Index(const catalog::index_oid_t oid, const ConstraintType constraint_type, IndexMetadata metadata)
```

Comment:
Why did we remove the `index_oid_t`? The `SQLTable` has its own `table_oid_t`?

Reply:
Objects should not need to reason about their own OID value as they should only be used to resolve logical "addresses" to physical ones.  `SqlTable` had the OID in its constructor removed as well.